### PR TITLE
Rj photogram industrial 2 added Associations, validations, and sample data

### PR DESCRIPTION
**Pull request by jrzricardo**

### CODE REVIEW

**Review by YoungSong99 — APPROVED** [review]
Looks great! It is clean code.

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Foreign Keys
+#
+#  fk_rails_...  (fan_id => users.id)
+#  fk_rails_...  (photo_id => photos.id)
+#
+class Like < ApplicationRecord
+  belongs_to :fan, class_name: "User", counter_cache: true
+  belongs_to :photo, counter_cache: true
+  
+  validates :fan_id, uniqueness: { scope: :photo_id, message: "has already like this photo" }
```
Comment (thierrychau) [inline app/models/like.rb:25]: Great job!

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :own_photos, class_name: "Photo", foreign_key: "owner_id"
+
+  has_many :comments, foreign_key: "author_id"
+
+  has_many :sent_follow_requests, foreign_key: :sender_id, class_name: "FollowRequest", dependent: :destroy
```
Comment (thierrychau) [inline app/models/user.rb:35]: Great job!

**Review by thierrychau — APPROVED** [review]
Very good, Ricardo. You are good to merge.

**Pull request merged**